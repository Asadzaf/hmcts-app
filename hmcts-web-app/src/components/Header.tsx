const Header = () => {
    return (
        <header className="govuk-header" data-module="govuk-header">
            <div className="govuk-header__container govuk-width-container">
                <div className="govuk-header__logo">
                    <a href="#" className="govuk-header__link govuk-header__link--homepage">
                        <svg
                            focusable="false"
                            role="img"
                            className="govuk-header__logotype"
                            xmlns="http://www.w3.org/2000/svg"
                            viewBox="0 0 148 30"
                            height="30"
                            width="148"
                            aria-label="GOV.UK">
                            <title>GOV.UK</title>
                        </svg>
                    </a>
                </div>
            </div>
        </header>
    )
}

export default Header;