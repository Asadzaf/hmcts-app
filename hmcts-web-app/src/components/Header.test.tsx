import '@testing-library/jest-dom';

import { render } from '@testing-library/react';
import Header from './Header';

test('renders Header correctly', () => {
  const { asFragment } = render(<Header />);
  expect(asFragment()).toMatchSnapshot();
});
