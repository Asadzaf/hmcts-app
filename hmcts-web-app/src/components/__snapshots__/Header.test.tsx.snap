// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header correctly 1`] = `
<DocumentFragment>
  <header
    class="govuk-header"
    data-module="govuk-header"
  >
    <div
      class="govuk-header__container govuk-width-container"
    >
      <div
        class="govuk-header__logo"
      >
        <a
          class="govuk-header__link govuk-header__link--homepage"
          href="#"
        >
          <svg
            aria-label="GOV.UK"
            class="govuk-header__logotype"
            focusable="false"
            height="30"
            role="img"
            viewBox="0 0 148 30"
            width="148"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              GOV.UK
            </title>
          </svg>
        </a>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
